package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "My-PhotoHub",
+  "version": "1.0.0",
+  "description": "JavaScript example starter project",
+  "main": "index.html",
+  "scripts": {
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html"
+  },
+  "dependencies": {
+    "parcel-bundler": "^2.0.0",
```

Comment:
Change this to `parcel` and use `^2.7.0` to reflect current version https://www.npmjs.com/package/parcel